impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -132,7 +132,8 @@ static void SetTextureLayout(
   }
   const auto& texture_vk = TextureVK::Cast(*texture);
 
-  if (attachment_desc.initialLayout == vk::ImageLayout::eGeneral) {
+  if (attachment_desc.initialLayout == vk::ImageLayout::eGeneral ||
```

Comment:
I think we should just unconditionally add the layout transition until we have evidence that its otherwise slow.